### PR TITLE
Added UNION testcases and auxilary code for running tests

### PR DESCRIPTION
**Pull request by ritwizsinha**

### CODE REVIEW

**Review by GuptaManan100 — APPROVED** [review]
This is great! 😀

File: go/vt/sqlparser/parse_test.go [hunk 1]
```diff
@@ -3492,3 +3495,173 @@ func BenchmarkParse3(b *testing.B) {
...
+func escapeNewLines(in string) string {
+	return strings.ReplaceAll(in, "\n", "\\n")
+}
+
+func testFile(t *testing.T, filename, tempDir string) {
+	t.Run(filename, func(t *testing.T) {
+		fail := false
+		expected := strings.Builder{}
+		for tcase := range iterateExecFile(filename) {
+			fmt.Println(tcase)
```
Comment (frouioui) [inline go/vt/sqlparser/parse_test.go:3541]: is this intentional?

**Review by frouioui — APPROVED** [review]
👍🏻

File: go/vt/sqlparser/parse_test.go (hunk 1 above)
Comment (GuptaManan100) [reply go/vt/sqlparser/parse_test.go:3541]: no it isn't! good catch! Thankyou

**Review by frouioui — APPROVED** [review]

**Review by harshit-gangal — APPROVED** [review]

**Pull request merged**